test-code/recode_road_class.R

```diff
@@ -0,0 +1,50 @@
+# Date      27/06/2022
+# Authour   James Hulse
+# Version   1.0.0
+#
+
+# Define recode_road_class function
+recode_road_class <- function(osm_sf) {
+  library("dplyr")
```

Comment:
This is pretty unconventional. For package code you don't need to attach the whole namespace. Your use of `dplyr::*()` means you don't need to load the package.

Reply:
I see - I had confused myself thinking you needed to specify ask some functions can be masked by other packages! will remove